rules.ini

```
@@ -193,6 +193,7 @@ cURL = curl(?:module|lib|d|-4|-ttv|-x64|64|_pluginw64_release)?\.(?:dll|exe|lib|
 DirectStorage = (?:^|/)dstorage\.dll$
 Discord = (?:^|/)(?:lib)?discord(?:|-rpc|_game_sdk)\.(?:dll|dylib|so)$
 EpicOnlineServices = (?:^|/)(?:lib)?eossdk
+Expat = (?:^|/)libexpat(?:w)?\.dll
```

```suggestion
Expat = (?:^|/)(?:lib|py)?expat
```
we can also look for the python lib